R/sim_gs_n.R

```diff
@@ -115,8 +117,8 @@
 #'   sample_size = 400,
 #'   enroll_rate = enroll_rate,
 #'   fail_rate = fail_rate,
-#'   test = wlr,
-#'   cutting = list(ia1 = ia1, ia2 = ia2, fa = fa),
+#'   tests = wlr,
```

Comment:
In both simtrial and gsDesign2, we use singular consistently. If you agree, would it be possible if we change from `tests` to `test` and `cuts` to `cut`?

Reply:
I completely agree. Consistency is a good reason to keep the argument names singular. I just pushed the update